sktime/forecasting/base/adapters/_neuralforecast.py

```diff
@@ -135,7 +135,6 @@ def _instantiate_model(self: "_NeuralForecastAdapter", fh: ForecastingHorizon):
 
         algorithm_instance = self.algorithm_class(
             fh,
-            alias=self.algorithm_name,
```

Comment:
Auto models don't suppport the alias parameter. I got error `AutoLSTM.__init__() got unexpected arguments alias`, so I had to remove it, will it affect the already implemented models, the tests are passing anyways

Reply:
This seems to be an issue with AutoLSTM, [the base class has that argument](https://github.com/Nixtla/neuralforecast/blob/404086c96ce30b4b2e5548191c73269da776e41d/neuralforecast/common/_base_auto.py#L76) and [some models use it](https://github.com/Nixtla/neuralforecast/blob/404086c96ce30b4b2e5548191c73269da776e41d/neuralforecast/auto.py#L280). We'll fix it but since it'll be on the next release it won't help you right now.

Reply:
Then I think it'll be better for us to not add interface to AutoLSTM one right now, as this change is breaking other estimators, see test failures for LSTM etc. in CI logs.

May be we can wait a few weeks (?) till next release of neuralforecast to ensure same argument in all models, and in the meanwhile @pranavvp16 you can contribute for a different one, e.g. `AutoTFT` as you have shown for TFT in Discord?

Reply:
We'll make a release this week, I'll make sure we include this change

Reply:
awesome! wiil update this PR as soon as neuralforecast receives the new release.